tests/steps/application_v2_steps.py

```diff
@@ -413,14 +369,15 @@ def wait_for_algod_transaction_processing_to_complete():
     time.sleep(0.5)
 
 
+# TODO: this needs to be modified/removed when v1 is no longer supported
```

Comment:
This should be a v2 step, so I think it should still be kept after v1 is _fully_ deprecated.